### PR TITLE
Implement hardcoded prototype of place suggestion feature

### PR DESCRIPTION
**Pull request by mffabrigas**
Upon load of the My Trips page, the website logs restaurants in 5000m radius of Mountain View, CA to browser console; these are listed by prominence by Places Library of Maps JavaScript API

### CODE REVIEW

**Comment by mffabrigas** [comment]
@googlebot I signed it!

**Comment by mffabrigas** [comment]
@googlebot

File: src/main/webapp/scripts/place-suggestions.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    radius: '5000',
+    type: ['restaurant']
+  };
+
+  service = new google.maps.places.PlacesService(map);
+  service.nearbySearch(request, callback);
+}
+
+function callback(results, status) {
+  if (status == google.maps.places.PlacesServiceStatus.OK) {
```
Comment (ptwu) [inline src/main/webapp/scripts/place-suggestions.js:23]: I think we should try our best to use `===` over `==` in all our JS code. It helps to reduce technical debt when we have a bunch of different types and yells at us if the two sides have some malformed type.

**Comment by ptwu** [comment]
Nice job fixing the conflict, but it seems that the `index.html` styles are not working / missing (see the background image)

**Comment by mffabrigas** [comment]
Changed the relative path url to the background in both CSS files for index.html (index.css and style.css) to account for change in file structure. On an aside, index.css and style.css seem redundant.

**Comment by ptwu** [comment]
> Changed the relative path url to the background in both CSS files for index.html (index.css and style.css) to account for change in file structure. On an aside, index.css and style.css seem redundant.

Hold on, the current master doesn't have a style.css. It looks like you started this branch from an older version (there's just index.css and constant-style.css now, which is more descriptive)

**Comment by ptwu** [comment]
> > Changed the relative path url to the background in both CSS files for index.html (index.css and style.css) to account for change in file structure. On an aside, index.css and style.css seem redundant.
> 
> Hold on, the current master doesn't have a style.css. It looks like you started this branch from an older version (there's just index.css and constant-style.css now, which is more descriptive)

Just checked the code for `style.css` and it looks like an older version of `index.css`. `style.css` can be deleted.

**Pull request merged**